nginx/examples.conf

```diff
@@ -20,18 +22,11 @@
 
 ###
 ## Example redirection
+## Note: Redirects are often much better implemented in PHP code.
+## See examples at https://seravo.com/docs/configuration/redirect/
```

Comment:
But this causes WP core to load even for static and trivial redirs > wasted CPU? Also, if we load WP, should we just recommend [Redirection](https://wordpress.org/plugins/redirection/) > users can have nice UI, and no risk of PHP syntax/logic errors?

From https://wordpress.stackexchange.com/a/26622

![7MhBq](https://user-images.githubusercontent.com/97568/82873278-eeb64c00-9f3c-11ea-822c-0449507342dc.png)

Reply:
We are recommending Redirect at https://seravo.com/docs/configuration/redirect/

Redirs done in PHP usually have 301 and get cached. Optimizing performance here is not the goal, but making more reliable and more specific redirects.